lms/services/jstor.py

```diff
@@ -2,27 +2,43 @@
 
 
 class JSTORService:
-    def __init__(self, settings):
-        self._settings = settings
+    def __init__(self, enabled, site_code):
```

Comment:
I don't remember why I was so set on passing the whole settings here. It event took some fighting with ApplicationSettings.

In any case, having the params explicitly declared is nicer than taking a random dict without known shape :+1: